### ISSUE TITLE
Web Sockets iOS support

### ISSUE THREAD

**Issue by adasiewiczr** [lightningkite/kommunicate#1]
Do you plan to add support for Web Sockets on iOS?

**Comment by UnknownJoe796** [comment]
No, and I'm planning to deprecate this library in favor of KTor, as it seems to be more complete anyways.
If you see a purpose in this repository, such that it should be kept around, please let me know and I'll see what I can do.
